bindata/kube-proxy/kube-proxy.yaml

```diff
@@ -0,0 +1,71 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: openshift-kube-proxy
+  namespace: openshift-kube-proxy
+  annotations:
+    kubernetes.io/description: |
+      This daemonset is the kubernetes service proxy (kube-proxy).
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+spec:
+  selector:
+    matchLabels:
+      app: kube-proxy
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: kube-proxy
+        component: network
+        type: infra
+        openshift.io/component: network
+    spec:
+      serviceAccountName: openshift-kube-proxy
+      hostNetwork: true
+      priorityClassName: system-node-critical
+      containers:
+      - name: kube-proxy
+        image: {{.KubeProxyImage}}
+        command:
+        - /usr/bin/hyperkube
+        - kube-proxy
+        # kube proxy args are injected by go
+        securityContext:
+          privileged: true
+        env:
+        # Tell kube-proxy to talk to the apiserver directly.
+        - name: KUBERNETES_SERVICE_PORT
+          value: "{{.KUBERNETES_SERVICE_PORT}}"
+        - name: KUBERNETES_SERVICE_HOST
+          value: "{{.KUBERNETES_SERVICE_HOST}}"
+        volumeMounts:
+        # Because we don't actually run iptables locally, but chroot in to the host
+        - mountPath: /host
+          name: host-slash
+          readOnly: true
+        terminationMessagePolicy: FallbackToLogsOnError
+        ports:
+        - name: healthz
+          containerPort: 10256
+        - name: metrics
+          containerPort: 9101
```

Comment:
If we allow these to be overridden then these should be template fields (and likewise in other places).
If we don't allow these to be overridden, then the code should be updated accordingly.

Reply:
good point.

Reply:
This is still not addressed

Reply:
Also a good point. fixed.